x/evidence/go.mod

```diff
@@ -154,3 +155,5 @@ require (
 )
 
 replace github.com/cosmos/cosmos-sdk => ../../.
+
+replace cosmossdk.io/x/params => ../params
```

Comment:
Should be removed.

Reply:
Yeah I had to re-run `go mod tidy` for all directories which I did by
```shell
find . -type f -name "go.mod" | xargs dirname | sort |uniq | while read F;do cd $F && go mod tidy;done
```